pkgs/tools/misc/turbo/default.nix

```diff
@@ -24,20 +24,22 @@
 , CoreFoundation
 }:
 let
-  version = "1.10.7";
+  version = "1.10.13";
   src = fetchFromGitHub {
     owner = "vercel";
     repo = "turbo";
     rev = "v${version}";
-    sha256 = "sha256-AkrwaaXUiFPZqOO1mX/1XBOZRFRtCdgI7glzdv8ZOfU=";
+    sha256 = "sha256-7bEHE/bHRVOXMP7+oo+4k8yn6d+LkXBi8JcDeR0ajww";
   };
 
   ffi = rustPlatform.buildRustPackage {
     pname = "turbo-ffi";
     inherit src version;
     cargoBuildFlags = [ "--package" "turborepo-ffi" ];
 
-    cargoHash = "sha256-j+r1irE0OGMfr9TAYhTOsFjBNzxjmF5/e7EebtshuG8=";
+    cargoLock = {
+      lockFile = "${src}/Cargo.lock";
```

Comment:
This doesn't work because it uses import from derivation, which isn't allowed inside nixpkgs. You can keep using `cargoHash` if it works, otherwise you would need to vendor the Cargo.lock into this directory and just use `./Cargo.lock`.

Reply:
Cheers updated